analyzers/tests/SonarAnalyzer.UnitTest/Rules/EmptyMethodTest.cs

```diff
@@ -78,45 +78,20 @@ public class EmptyMethodTest
         public void EmptyMethod_VB() =>
             Verifier.VerifyAnalyzer(@"TestCases\EmptyMethod.vb", new VB.EmptyMethod());
 
-        [DataTestMethod]
-        [DataRow(ProjectType.Product)]
-        [DataRow(ProjectType.Unknown)]
-        public void EmptyMethod_WithVirtualOverride_RaisesIssueForMainAndUnknownProject_CS(ProjectType projectType) =>
-            Verifier.VerifyAnalyzer(@"TestCases\EmptyMethod.OverrideVirtual.cs",
-                new CS.EmptyMethod(),
-                options: ParseOptionsHelper.FromCSharp8,
-                additionalReferences: null,
-                sonarProjectConfigPath: TestHelper.CreateSonarProjectConfig(
-                    nameof(EmptyMethod_WithVirtualOverride_RaisesIssueForMainAndUnknownProject_CS),
-                    projectType));
+        [TestMethod]
+        public void EmptyMethod_WithVirtualOverride_RaisesIssueForMainProject_CS() =>
+            Verifier.VerifyAnalyzer(@"TestCases\EmptyMethod.OverrideVirtual.cs", new CS.EmptyMethod());
 
         [TestMethod]
         public void EmptyMethod_WithVirtualOverride_DoesNotRaiseIssuesForTestProject_CS() =>
-            Verifier.VerifyNoIssueReported(@"TestCases\EmptyMethod.OverrideVirtual.cs",
-                new CS.EmptyMethod(),
-                ParseOptionsHelper.FromCSharp8,
-                sonarProjectConfigPath: TestHelper.CreateSonarProjectConfig(
-                    nameof(EmptyMethod_WithVirtualOverride_DoesNotRaiseIssuesForTestProject_CS),
-                    ProjectType.Test));
+            Verifier.VerifyNoIssueReportedInTest(@"TestCases\EmptyMethod.OverrideVirtual.cs", new CS.EmptyMethod());
```

Comment:
We already have this `VerifyNoIssueReportedInTest` method that we can use directly. For the rule, it's not important how it was detected as a test.

If really needed, I can do the SonarProjectConfig magic inside `VerifyNoIssueReportedInTest`, but it will behave the same.